### DISCUSSION TITLE
Release v2.1.3 版本更新预告

### DISCUSSION THREAD

**Discussion by bao-cn** [Lunova-Studio/DarkMode2#40] [Announcements]
下个版本将迎来代码的重构，软件85%的代码将得到优化。
下个版本更新内容如下：
1. 支持多线程操作。软件会从单线程应用转变为多线程应用，对于程序意外的卡死或者程序的稳定性会有显著提高；
2. 优化快捷键#30。设置中上线快捷键设置，可以自定义快捷键或者取消快捷键；
3. 右键菜单上线快捷切换深浅色功能（实验室）#26。将会给右键菜单添加切换深浅色的快捷功能，支持一键切换深浅色，该状态下自动切换的功能将临时禁用，下一次启动电脑时自动恢复；
4. 快捷键切换深浅色功能（实验室）#29。设置中同样支持设置快捷键一键切换深浅色，切换模式与右键菜单相同；
5. 优化注册表异常处理机制#32。
6. 修复：自定义时间越界Bug #33。

**Comment by bao-cn** [comment]
内存占用不稳定的情况将得到显著改善

**Comment by bfgxp** [comment]
非常期待新版
